validator/specs/lib/validateTilesetSpec.js

```diff
@@ -131,10 +131,77 @@ describe('validateTileset', function() {
             }), done).toResolve();
     });
 
+    it('returns error message when a content\'s box type boundingVolume is not within it\'s tile\'s sphere type boundingVolume [invalid bounding boxes]', function(done) {
+        var tileBoundingVolume = {
+            sphere: [
+                0, 0, 0,
+                1
+            ]
+        };
+        var contentBoundingVolume = {
+            box: [
+                0, 0, 0,
+                1, 0, 0,
+                0, 0.5, 0,
+                0, 0, 0.7
+            ]
+        };
+        var tileset = createSampleTileset(tileBoundingVolume, contentBoundingVolume);
+        expect(validateTileset(tileset)
+            .then(function(message) {
+                expect(message).toBe('content box [' + contentBoundingVolume.box + '] is not within tile box [' + tileBoundingVolume.sphere + ']');
+            }), done).toResolve();
+    });
+
+    it('returns error message when a content\'s box type boundingVolume is not within it\'s tile\'s sphere type boundingVolume [valid bounding boxes]', function(done) {
```

Comment:
Like in the other PR, fix description so it is clearer that this test is valid, it doesn't return an error message.